src/gui/about_imp.h

```diff
@@ -36,6 +36,11 @@
 #include "base/utils/misc.h"
 #include "base/unicodestrings.h"
 
+#define SIXTYFOUR ""
```

Comment:
Since this is a header file, could you replace the macro with a regular constant, please? Moreover, the bitness string is not needed outside of the class constructor, there is no need to pollute the global scope. E.g.:
```C++
lb_name->setText(QLatin1String("<b><h2>qBittorrent ")
			+ QLatin1String(VERSION) + (sizeof(void*) == 8 ? QLatin1String(" (64 bit)") : QLatin1String())
			+ QLatin1String("</h2></b>"));
```